src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from typing import Optional
+from enum import Enum
+import errno
+import json
+import requests
+
+from requests.auth import AuthBase
+from mgr_module import CLICommand, CLIWriteCommand, CLIReadCommand, HandleCommandResult
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException, ScopeNotInRole
+from ..rest_client import RestClient
+from ..model.feedback import Feedback
+
+class config:
+    url = 'tracker.ceph.com'
+    port = 443
```

Comment:
I would use dataclasses for this but probably is an overkill and out of reach. (it is just a remark for the future)